devlib/platform/__init__.py

```diff
@@ -78,7 +78,16 @@ def _set_core_clusters_from_core_names(self):
 
     def _set_model_from_target(self, target):
         if target.os == 'android':
-            self.model = target.getprop('ro.product.model')
+            try:
+                self.model = self.getprop(prop='ro.product.device')
```

Comment:
Should this not be `target.getprop()`?

Reply:
Yup it should. Thanks for catching my stupidity. Should be fixed now.